.github/workflows/golang-build-test.yml

```diff
@@ -69,6 +69,40 @@ jobs:
           name: Unit Test Results
           path: "${{ github.workspace }}/src/go/test_src_go.xml"
 
+  test_src_go_qemu:
+    needs: pre_job_src_go_determinator
+    if: ${{ needs.pre_job_src_go_determinator.outputs.should_skip == 'false' }}
+    strategy:
+      matrix:
+        go-version: [1.17.x]
+        arch: [386, arm, arm64]
```

Comment:
not sure if 386 is necessary, but i guess it doesn't hurt! :)

Reply:
My logic was that if something failed in the `arm` / `arm64` jobs, it would be nice to have a sanity check x86 build to confirm it's not QEMU or infra related. The `386` was arbitrary... Don't mind dropping though! *shruggie*